src/steps/generic.rs

```diff
@@ -175,6 +175,18 @@ pub fn run_rustup(base_dirs: &BaseDirs, run_type: RunType) -> Result<()> {
     run_type.execute(&rustup).arg("update").status_checked()
 }
 
+pub fn run_juliaup(base_dirs: &BaseDirs, run_type: RunType) -> Result<()> {
+    let juliaup = utils::require("juliaup")?;
+
+    print_separator("juliaup");
+
+    if juliaup.canonicalize()?.is_descendant_of(base_dirs.home_dir()) {
+        run_type.execute(&juliaup).args(["self", "update"]).check_run()?;
+    }
+
+    run_type.execute(&juliaup).arg("update").check_run()
```

Comment:
Could you please replace chech_run with status_checked(). Ex.: https://github.com/topgrade-rs/topgrade/blob/c83c850546d00c2d73b47263ed6c622e72463afe/src/steps/generic.rs#L175

Reply:
Done. Thanks.